db/columns/operations/select.py

```diff
@@ -8,6 +9,14 @@
 from db.utils import execute_statement, get_pg_catalog_table
 
 
+def get_column_description(oid, attnum, engine):
+    conn_str = str(engine.url)
+    with psycopg.connect(conn_str) as conn:
+        row = conn.execute(f"SELECT col_description({oid}, {attnum})").fetchone()
+        description = row[0]
+    return description
```

Comment:
I'll see about doing that.

Reply:
> At first glance, this seems like it would be worth making a DB layer wrapper for this so it fits with the rest of our machinery. What do you think?

Could you expand on this a bit more? I understand the "general 'run a function and return the cursor' python wrapper" idea, but I'm not sure about your first suggestion.

Reply:
The first suggestion would be to create something like `msar.col_description(tab_id oid, col_id integer)` on the DB that's just a thin wrapper around the PostgreSQL `col_description` . The value of that is that it'd be callable from Python using the same `execute_msar_func_with_engine` _Python_ wrapper that we use elsewhere. I'm trying to keep all new calls to the database funneled through a pretty controlled spot right now (the `db.connection` module), since we'll need to refactor that eventually and I don't want to hunt through every `conn.execute` in the codebase at that point.